docs/api.md

```diff
@@ -212,6 +215,76 @@ nos.send({ asset: GAS, amount, receiver })
     .catch((err) => alert(`Error: ${err.message}`));
 ```
 
+### `getPublicKey`
+The `getPublicKey` provides the public key of the currently authenticated account. It does not require the user to grant permission.
+
+#### Parameters
+None.
+
+#### Returns
+**string** - The public key of the currently signed in user.
+
+#### Example
+```javascript
+const nos = window.NOS.V1;
+
+nos.getPublicKey()
+  .then((publicKey) => alert(`Public Key: ${publicKey}`))
+  .catch((err) => alert(`Error: ${err.message}`));
+```
+
+### `encrypt`
+The `encrypt` function allows you to encrypt arbitrary data for another user (you will need his public key, please see `getPublicKey`). It does not require the user to grant permission.
+
+#### Parameters
+* `config` **object** - The config options to perform this operation.
+* `config.publicKey` **string** - The public key of the recipient account.
+* `config.data` **string | Buffer** - The data to encrypt.
+
+#### Returns
+**object** - object containing encrypted data and values needed for decryption (`iv`, `mac`, `data`)
+
+#### Example
+```javascript
+const nos = window.NOS.V1;
+
+// for the test data below WIF of the sender account is KxDgvEKzgSBPPfuVfw67oPQBSjidEiqTHURKSDL1R7yGaGYAeYnr
+const recipientPublicKey = '031a6c6fbbdf02ca351745fa86b9ba5a9452d785ac4f7fc2b7548ca2a46c4fcf4a';
+const data = 'some text';
+
+nos.encrypt({ recipientPublicKey, data })
+  .then(({ iv, mac, data }) => alert(`iv: ${iv}\nmac: ${mac}\ndata: ${data}`))
+  .catch((err) => alert(`Error: ${err.message}`));
+```
+
+### `decrypt`
+The `decrypt` function allows you to decrypt previously encrypted data for this user (you will need public key of the sender account, please see `getPublicKey`). It does not require the user to grant permission.
+
+#### Parameters
+* `config` **object** - The config options to perform this operation.
+* `config.publicKey` **string** - The public key of the sender account.
+* `config.iv` **string** - The IV received during encryption.
+* `config.mac` **string** - The MAC received during encryption.
+* `config.data` **string** | **Buffer** - The data to decrypt.
+
+#### Returns
+**Buffer** - decrypted data
```

Comment:
@trueinsider This is the return value of type `Buffer` I was referring to.

Reply:
oh, you right. but it actually returns Buffer.

inside decrypt:
>  return aes256CbcDecrypt(ivBuffer, encryptionKey, dataBuffer);

and inside aes256CbcDecrypt:
> return Buffer.concat([firstChunk, secondChunk]);